### DISCUSSION TITLE
Example Invetory?

### DISCUSSION THREAD

**Discussion by tomasnorre** [deployphp/deployer#2201] [General]
Hi,

What is your idea of the section Separate inventory files? 
https://deployer.org/docs/examples/inventory.html#separate-inventory-files

To me it looks like it's covered in: 
https://deployer.org/docs/hosts.html#inventory-file

But could be that I'm wrong about the purpose of that section. I just started to look into deployer and it looks really promising regards my requirements. 

I would be happy to help update the docs for this section if needed.

**Comment by antonmedv** [comment]
Hello! 

Docs actually outdated and Im working on new better docs. 
And in soon inventory will be gone with replacement of import() function.
